opencti-platform/opencti-front/src/private/components/common/form/CreatorField.tsx

```diff
@@ -74,49 +89,73 @@ const CreatorField: FunctionComponent<CreatorFieldProps> = ({
         const NewCreators = (
           (data as CreatorFieldSearchQuery$data)?.members?.edges ?? []
         ).map((n) => ({
-          label: n?.node.name,
+          label: n?.node.name ?? t_i18n('Unknown'),
           value: n?.node.id,
+          extra: getExtraFromNode(n?.node),
         }));
-        const templateValues = [...creators, ...NewCreators];
+        const templateValues = [...creatorOptions, ...NewCreators];
         // Keep only the unique list of options
         const uniqTemplates = templateValues.filter((item, index) => {
           return (
             templateValues.findIndex((e) => e.value === item.value) === index
           );
         });
-        setCreators(uniqTemplates);
+        setCreatorOptions(uniqTemplates);
       });
   };
-
   return (
     <div style={{ width: '100%' }}>
       <Field
         component={AutocompleteField}
         name={name}
         textfieldprops={{
           variant: 'standard',
-          label: t_i18n(label),
-          helperText: helpertext,
+          label,
           onFocus: searchCreators,
         }}
+        disableClearable
         onChange={onChange}
         style={containerStyle}
         noOptionsText={t_i18n('No available options')}
-        options={creators}
-        isOptionEqualToValue={isOptionEqualToValue}
+        options={creatorOptions}
+        isOptionEqualToValue={(option: CreatorOption, selected: CreatorOption) => option.value === selected.value}
```

Comment:
Having `isOptionEqualToValue` as an external props is unnecessary, as it's always the same in all cases.
It was also badly typed and used, and resulted in react  warnings all around.

Now it's the right signature.